Src/Xceed.Wpf.Toolkit/Themes/Generic.xaml

```diff
@@ -18,50 +18,45 @@
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="DropDownButton/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="IconButton/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="Magnifier/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="MessageBox/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="DateTimeUpDown/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="DateTimePicker/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="MultiLineTextEditor/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="NumericUpDown/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="Pie/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="RangeSlider/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="SplitButton/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="PropertyGrid/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="RichTextBoxFormatBar/Themes/Generic.xaml" />
-
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="TimePicker/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="TimeSpanUpDown/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="WatermarkComboBox/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="WatermarkTextBox/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="Wizard/Themes/Generic.xaml" />
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="Zoombox/Themes/Generic.xaml" />
-
-      <core:VersionResourceDictionary AssemblyName="DotNetProjects.Wpf.Extended.Toolkit" SourcePath="TokenizedTextBox/Themes/Generic.xaml" />
-    </ResourceDictionary.MergedDictionaries>
+      <core:VersionResourceDictionary AssemblyName="Xceed.Wpf.Toolkit" SourcePath="Themes/Generic/Brushes.xaml" />

```

Comment:
and here

Reply:
I overlooked these. And also create a simple app to test, and you are right. So I made the changes, and now it works correctly :)